safe/definitions/extra_keywords.py

```diff
@@ -113,8 +128,46 @@
         'extent of the event.')
 }
 
+extra_keyword_earthquake_source = {
+    'key': 'earthquake_source',
+    'name': tr('Source'),
+    'description': tr(
+        'Source of the earthquake, it can be initial or post-processed '
+        '(data-informed).'),
+    'type': unicode,
+    'options': [
+        {
+            'key': 'initial',
+            'name': tr('Initial'),
+            'description': tr(
+                'This shakemap is created in 5 minutes after the earthquake. '
```

Comment:
I would also remove reference to realtime in InaSAFE desktop. Can we change this 5 minutes? Is not specific to realtime? Maybe `This shakemap is created just after an EQ`. ? I'm not quite happy with what I'm proposing too.

Reply:
I changed it to:
> This shakemap is created in just after the earthquake '
                'without any extra processing happens.

since the main point is extra processing's existence.